generator/src/routes/+page.svelte

```diff
@@ -553,6 +553,39 @@
     align-items: center;
     flex-direction: row;
     cursor: pointer;
+    position: relative;
+    &:hover .tooltip {
+      visibility: visible;
+      opacity: 1;
+    }
+    .tooltip {
+      position: absolute;
+      left: calc(100% + 0.3125rem);
+      visibility: hidden;
+      color: #101920;
+      font-size: 0.75rem;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 1.125rem;
+      padding: 0.5rem 0.75rem;
+      border-radius: 0.3125rem;
+      width: 15rem;
+      box-shadow: 0px 0px 4px 0px rgba(0, 0, 0, 0.15);
+      opacity: 0;
+      transition: opacity 1s;
```

Comment:
Super slow transitions are mostly annoying
```suggestion
      transition: opacity 0.15s;
```